### PR TITLE
Adding a resources page

### PR DESCRIPTION
**Pull request by sujee**
## Why are these changes needed?

Placeholder for listing growing number of resources

### CODE REVIEW

**Comment by shahrokhDaijavad** [comment]
I do like the idea of having this resource page, but I saw @Bytes-Explorer 's reservations about doing this. I discussed it with Nirmit this morning and his general direction is not to make the README page longer than what it needs to be and by not having a separate resource page for these links, the README keeps getting longer. In any case, let's discuss the pros/cons in our Tuesday meeting, before approving or not.

**Review by shahrokhDaijavad — CHANGES_REQUESTED** [review]
Actual changes are not needed, but I put it in "request change" mode, so it doesn't get approved accidentally, before we discuss further (see my comment).

**Comment by sujee** [comment]
> I do like the idea of having this resource page, but I saw @Bytes-Explorer 's reservations about doing this. I discussed it with Nirmit this morning and his general direction is not to make the README page longer than what it needs to be and by not having a separate resource page for these links, the README keeps getting longer. In any case, let's discuss the pros/cons in our Tuesday meeting, before approving or not.

I am also in favor of compact top level README.  Let's discuss.

**Comment by Bytes-Explorer** [comment]
I dont have a strong opinion here. My point is that it should be easy for the user to find the information and usually I have seen repos have this information in the top level readme. @touma-I Aanchal - pls share your suggestions too.

**Review by shahrokhDaijavad — APPROVED** [review]
LGTM. @touma-I, we discussed this today, and unless you have a strong opinion against it, let's merge.

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ The goal is to offer high-level APIs for developers to quickly get started in wo
 - [Scaling transforms from laptop to cluster](#laptop_cluster)
 - [Repository Use and Navigation](doc/repo.md)
 - [How to Contribute](CONTRIBUTING.md)
-- [Talks and Papers](#talks_papers)
+- [Resources (papers, talks, presentations and tutorials)](resources.md)
 - [Citations](#citations)
```
Comment (touma-I) [inline README.md:35]: Need to first sync the fork with upstream and then update the README.md to take into account the changes that were done to the upstream file.

**Review by touma-I — CHANGES_REQUESTED** [review]
@sujee Need to sync with upstream and update branch with latest before editing latest.

**Comment by sujee** [comment]
> @sujee Need to sync with upstream and update branch with latest before editing latest.

done.  please check again. thx

**Review by shahrokhDaijavad — APPROVED** [review]
LGTM. Thanks @sujee .

**Review by touma-I — APPROVED** [review]

**Pull request merged**